frontend/components/App/FishbowlPreJoin/index.tsx

```diff
@@ -64,22 +63,22 @@ const FishbowlPreJoin: React.FC = () => {
     localTracks.current = [];
   };
 
-  const handleParentClick = event => {
-    if (event.target.id !== 'config-button') {
+  const handleParentClick = (event: React.MouseEvent<HTMLDivElement>): void => {
+    if ((event.target as HTMLDivElement).id !== 'config-button') {
```

Comment:
this is redundant, since on the previous line you are already declaring it generic with HTmlDivElement

Reply:
It does not, as not al event targets are HTML Elements. Check [this thread](https://github.com/Microsoft/TypeScript/issues/29540) about this issue